app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,20 @@
       <%= render "partials/primary_nav" %>
     </div>
 
+    <div class="flash-alerts">
```

Comment:
Nice–this looks better! There are different styles for the different types of notices, and we only have the flash message logic once.

The two types of notices share a basic template though, and I imagine we want them to grow and change together. Can you think of a way to let them share a template?

At the least, what do you think of putting the logic in a partial? That'll let the application layout stay a high-level template (the level of detail in these lines is different than the rest of the file), and it'll keep this flash message logic in one contained location.